adafruit_sht31d.py

```diff
@@ -276,45 +285,45 @@ def mode(self, value):
         self._mode = value
 
     @property
-    def repeatability(self):
+    def repeatability(self) -> Literal["High", "Medium", "Low"]:
         """
         Repeatability
         Allowed values are the constants REP_*
         """
         return self._repeatability
 
     @repeatability.setter
-    def repeatability(self, value):
+    def repeatability(self, value: Tuple[str, bool, int]) -> None:
```

Comment:
The type annotation for `value` should be the same as the return of the `repeatability` getter.